common/src/main/java/org/terracotta/angela/common/distribution/Distribution107Controller.java

```diff
@@ -72,10 +74,21 @@ public Distribution107Controller(Distribution distribution) {
     super(distribution);
   }
 
+  private void installCustomLogging(File workingDir) {
+      try {
+        if (!Files.exists(workingDir.toPath().resolve("logback-test.xml"))) {
+          Files.copy(this.getClass().getResourceAsStream("/tc-logback.xml"), workingDir.toPath().resolve("logback-test.xml"));
```

Comment:
unfortunately this will fail when running a distributed test: the Tsa instance is running in the test JVM, then creates a remote Terracotta Instance (which then creates the DistributionController), therefore if the test is not fully localhost, the remote DistributionController will not find the File which exists only on the machine which executed the test

Reply:
the file is included in angela-common.  Isn't that included with angela-agent-lib?